.kitchen.yml

```diff
@@ -47,3 +47,34 @@ suites:
   - name: ftp
     run_list:
       - recipe[osl-jenkins::ftp]
+  # In order properly test Github integration, you must export the GITHUB_USER
+  # and GITHUB_TOKEN env vars in your shell before you converge this suite.
```

Comment:
The username+token combo is used for `git pull`/`git push`, not for the API, which just requires the token.  Just passing a username is easier than worrying about generating an ssh keypair too.